### ISSUE TITLE
Opening terminal in container fails

### ISSUE THREAD

**Issue by gertjanklein** [intersystems-community/vscode-objectscript#1133]
Running on Windows 10. Using docker desktop v4.17.1, InterSystems ObjectScript extension v2.8.0.

Clicking on the ObjectScript connection button on the toolbar opens a 'menu dropdown'. Selecting "Open Terminal in Docker" from that menu gives this error:

`The terminal process failed to launch: A native exception occurred during launch (Cannot create process, error code: 193).`

To reproduce, checkout [this example](https://github.com/intersystems-community/objectscript-docker-template), run `docker-compose build` and `docker-compose up `, open VS code, and attempt to open a terminal as described.

The "Pty Host" output shows:

`2023-04-11 16:48:13.941 [warning] Shell integration cannot be enabled for executable "C:\Program Files\Docker\Docker\resources\bin\docker" and args ["compose","-f","docker-compose.yml","exec","iris","/bin/bash","-c","[ -f /tmp/vscodesession.pid ] && kill $(cat /tmp/vscodesession.pid) >/dev/null 2>&1 ; echo $$ > /tmp/vscodesession.pid;\n        $(command -v ccontrol || command -v iris) session $ISC_PACKAGE_INSTANCENAME -U USER"]`

If I run this command in a terminal it works:

`"C:\Program Files\Docker\Docker\resources\bin\docker" compose -f docker-compose.yml exec iris /bin/bash -c "[ -f /tmp/vscodesession.pid ] && kill $(cat /tmp/vscodesession.pid) >/dev/null 2>&1 ; echo $$ > /tmp/vscodesession.pid; $(command -v ccontrol || command -v iris) session $ISC_PACKAGE_INSTANCENAME -U USER"`

(Note that I removed a '\n' here, but I doubt somehow this is the reason.)  I am not sure how to debug this further. An internet search suggests that a common reason for error code 193 is whitespace in the executable path, which I indeed have.

**Comment by isc-bsaviano** [comment]
I was able to reproduce this on my windows machine but since I neither use Windows nor this feature it's going to take me some time to figure it out.

**Comment by gertjanklein** [comment]
Thanks for looking into this. I can add that I tried the various suggestions on the [Open Help](https://code.visualstudio.com/docs/supporting/troubleshoot-terminal-launch) page, i.e. virus scanner exception, checking console mode, etc.  I also attempted to add quotes around the executable in the ObjectScript extension source, but either it didn't help or I did it wrong. (I don't speak TypeScript, and I don't understand enough of that code.)
